src/education/actions.js

```diff
@@ -0,0 +1,23 @@
+import { redirect } from "react-router-dom";
+
+export async function postAction({ request }) {
+  const formData = await request.formData();
+  const data = Object.fromEntries(formData);
+
+  const startDate = new Date(data.start_date);
+  const startMonth = startDate.toLocaleString("default", { month: "long" });
+  const startYear = startDate.getFullYear();
+  const endDate = new Date(data.end_date);
+  const endMonth = endDate.toLocaleString("default", { month: "long" });
+  const endYear = endDate.getFullYear();
+
+  await fetch("/resume/education", {
```

Comment:
@dtemir how are we hosting the app? Is the backend serving the frontend as static files or is the frontend gonna be hosted somewhere else?

Reply:
let's assume everything is running locally for now and there is no hosting in sight